apps/aws-notebook/samples/ecs/fargate/vpn/build_openvpn_config.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+eval "$(jq -r '@sh "VPN_DNS=\(.vpn_server_dns)"')"
+
+# The VPN_DNS we get from terraform will have the prefix '*.', so we trim it. 
+# Ref: https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/cvpn-getting-started.html#:~:text=Locate%20the%20line%20that%20specifies%20the%20Client%20VPN%20endpoint%20DNS%20name,asdfa.cvpn%2Dendpoint%2D0102bc4c2eEXAMPLE.prod.clientvpn.us%2Dwest%2D2.amazonaws.com
+cat > ./generated/vpn_client_config.ovpn << EOF
+client
+dev tun
+proto udp
+remote random.${VPN_DNS:2:${#VPN_DNS}} 443
+remote-random-hostname
+resolv-retry infinite
+nobind
+remote-cert-tls server
+cipher AES-256-GCM
+verb 3
+route-nopull
+route 10.0.0.0 255.255.0.0
```

Comment:
Please pull route from server, do not hard-code on the config.
We can generate the config using terraform, here's example: https://github.com/crafting-dev/sandbox-deploy/blob/master/tf/modules/sandbox/aws/vpn/client_config.tf
The benefit is we can consistently using `terraform output -raw KEY` to fetch vpn config, even don't need to save it.

Here's example how to use the output without saving it: https://github.com/crafting-dev/sandbox-deploy/blob/master/scripts/pack/data/selfhost/skeleton/scripts/connect-vpn.sh

Reply:
If we don't set route manually, the vpn client will set the default route of our sandbox through the vpn tunnel interface. It will result that  all traffic will go through the vpn and if the vpn can't access internet, the sandbox will also have no access to internet @easeway